autonomi/src/client/utils.rs

```diff
@@ -146,19 +150,21 @@ impl Client {
         let put_cfg = PutRecordCfg {
             put_quorum: Quorum::One,
             retry_strategy: Some(RetryStrategy::Balanced),
-            use_put_record_to: Some(vec![storing_node]),
+            use_put_record_to: Some(storing_nodes), // CODE REVIEW: do we put to all payees or just one?
```

Comment:
Same here, this would be a good optimization down the line but maybe in the put_record code layer?